### PR TITLE
Id Domain Content, Domain nav styles

### PR DESCRIPTION
**Pull request by nbuonin**
This PR:
- Fills in some content for the first learning domain
- Updates some styles to the vertical nav in the domain pages
- Moves and renames the reset button to the footer

### CODE REVIEW

File: content/domains/identify/demonstrate-awareness-of-opioid-risk/_index.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 date: 2019-05-13
 layout: learning-objective
 type: learning-objective
 weight: 2
 ---
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
+The 40-year old patient 1a. returns to the clinic two weeks after the emergency
```
Comment (sdreher) [inline content/domains/identify/demonstrate-awareness-of-opioid-risk/_index.md]: 1a?

File: content/domains/identify/screen-for-substance-disorders/_index.md [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 type: learning-objective
 weight: 1
 ---
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
+A 40-year old male patient presents to the clinic in acute pain. While the
+dental assistant begins to take diagnostic radiographs the patient repeatedly
+doses off. When you arrive to the operatory the patient finishes a conversation
```
Comment (sdreher) [inline content/domains/identify/screen-for-substance-disorders/_index.md]: dozes off

File: content/domains/identify/demonstrate-awareness-of-opioid-risk/pre-assessment.md [hunk 3]
```diff
@@ -4,10 +4,7 @@ date: 2019-05-13
...
 weight: 1
-q1: ["SENTINAL_0", "SENTINAL_1", "SENTINAL_2", "SENTINAL_3", "SENTINAL_4"]
+q1: ["Strongly disagree", "Disagree", "Neither agree nor disagree", "Agree", "Strongly agree"]
 ---
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
-
+I’m familiar with screening (PDMP) and assessment (social history
```
Comment (sdreher) [inline content/domains/identify/demonstrate-awareness-of-opioid-risk/pre-assessment.md]: Remove the question mark. This should present as a statement not a question.

File: content/domains/identify/screen-for-substance-disorders/_index.md [hunk 4]
```diff
@@ -1,11 +1,21 @@
...
+on the telephone then begins to hold his face reporting excruciating pain.
```
Comment (sdreher) [inline content/domains/identify/screen-for-substance-disorders/_index.md]: telephone, (add a comma)

File: content/domains/identify/screen-for-substance-disorders/post-assessment.md [hunk 5]
```diff
@@ -4,10 +4,21 @@ date: 2019-05-13
...
-q1: ["SENTINAL_0", "SENTINAL_1", "SENTINAL_2", "SENTINAL_3", "SENTINAL_4"]
+q1: ["Strongly disagree", "Disagree", "Neither agree nor disagree", "Agree", "Strongly agree"]
 ---
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
+A 40-year old male patient presents to the clinic in acute pain. While the
+dental assistant begins to take diagnostic radiographs the patient repeatedly
+doses off. When you arrive to the operatory the patient finishes a conversation
```
Comment (sdreher) [inline content/domains/identify/screen-for-substance-disorders/post-assessment.md]: dozes

File: content/domains/identify/screen-for-substance-disorders/post-assessment.md [hunk 6]
```diff
@@ -4,10 +4,21 @@ date: 2019-05-13
...
+on the telephone then begins to hold his face reporting excruciating pain.
```
Comment (sdreher) [inline content/domains/identify/screen-for-substance-disorders/post-assessment.md]: telephone, (add comma)

File: content/domains/identify/screen-for-substance-disorders/post-assessment.md [hunk 7]
```diff
@@ -4,10 +4,21 @@ date: 2019-05-13
...
+The patient’s wisdom tooth is partially impacted with no caries, no pathology,
+no inflammation of gingiva. As the patient begins to describe his symptoms; you
+notice some of his speech is slurred, and he becomes easily irritated with
+follow-up questions. When you begin your intraoral examination, the patient has
+poor oral hygiene with heavy plaque, heavy calculus build-up and a poor odor.
+You decide to give the patient a prescription for Motrin 600mg and provide him
+with a referral to have the symptomatic tooth extracted at an oral surgeon’s
+office close to your location and schedule a follow-up appointment in 2 weeks.
+
+How comfortable are you addressing with the patient conducting a screening to
```
Comment (sdreher) [inline content/domains/identify/screen-for-substance-disorders/post-assessment.md]: super awkward sentence. (repeated in the pre-assessment.md too). Maybe something like this?

How comfortable are you talking with the patient about a screening to determine risk or presence of substance use disorder?

File: content/domains/identify/screen-for-substance-disorders/pre-assessment.md [hunk 8]
```diff
@@ -4,10 +4,7 @@ date: 2019-05-13
...
 weight: 1
-q1: ["SENTINAL_0", "SENTINAL_1", "SENTINAL_2", "SENTINAL_3", "SENTINAL_4"]
+q1: ["Strongly disagree", "Disagree", "Neither agree nor disagree", "Agree", "Strongly agree"]
 ---
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
-
+How comfortable are you addressing with the patient conducting a screening to
```
Comment (sdreher) [inline content/domains/identify/screen-for-substance-disorders/pre-assessment.md]: see above for alternate wording.

File: layouts/partials/footer.html [hunk 9]
```diff
@@ -1,6 +1,32 @@
...
+            </div>
+        </div>
     </div>
 </footer>
+<!-- Modal -->
+<div class="modal fade" id="resetModal" tabindex="-1" role="dialog" aria-labelledby="resetModalLabel" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="resetModalLabel">Refresh Your Data</h5>
```
Comment (sdreher) [inline layouts/partials/footer.html]: Change "Refresh" to "Reset"

File: content/domains/identify/screen-for-substance-disorders/post-assessment.md (hunk 7 above)
Comment (sdreher) [reply content/domains/identify/screen-for-substance-disorders/post-assessment.md]: Also...the actual page has this sentence matched with: Strongly agree, etc. Likely this should change to a statement: I am comfortable talking with the patient about a screening to determine risk or presence of substance use disorder.

File: content/domains/identify/demonstrate-awareness-of-opioid-risk/_index.md [hunk 10]
```diff
@@ -1,11 +1,19 @@
...
+evaluation for a partially impacted wisdom tooth led to analgesic therapy and a
+referral to the oral surgeon for extraction of symptomatic tooth. When you
+enter the operatory the patient introduces himself to you again, not
```
Comment (sdreher) [inline content/domains/identify/demonstrate-awareness-of-opioid-risk/_index.md]: operatory, (add comma)

File: content/domains/identify/demonstrate-awareness-of-opioid-risk/post-assessment.md [hunk 11]
```diff
@@ -4,10 +4,19 @@ date: 2019-05-13
...
+q1: ["Strongly disagree", "Disagree", "Neither agree nor disagree", "Agree", "Strongly agree"]
 ---
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
+The 40-year old patient 1a. returns to the clinic two weeks after the emergency
+evaluation for a partially impacted wisdom tooth led to analgesic therapy and a
+referral to the oral surgeon for extraction of symptomatic tooth. When you
+enter the operatory the patient introduces himself to you again, not
```
Comment (sdreher) [inline content/domains/identify/demonstrate-awareness-of-opioid-risk/post-assessment.md]: operatory, (add comma)

File: content/domains/identify/importance-of-dental-providers-role/post-assessment.md [hunk 12]
```diff
@@ -4,10 +4,21 @@ date: 2019-05-13
...
+care for pregnancy gingivitis. Last year she was in a car accident and had to
+have surgery on her rotator cuff. You notice during this examination that her
+oral hygiene and home care is not as thorough as it once was. Due to your
+long-standing rapport with her, she divulges with you that she suffered from
+postpartum depression and the hydrocodone she was prescribed post shoulder
+surgery made her feel much better. She asks you if by chance you could
+prescribe her hydrocodone because of your history with her and how comfortable
+she has become with you.
+
+How prepared do you feel as a dental clinician to address the overall health
```
Comment (sdreher) [inline content/domains/identify/importance-of-dental-providers-role/post-assessment.md]: per the rest of the interactive, change to a statement: I am prepared as a dental clinician to address the overall health (including non-dental) of my patient population.

File: content/domains/identify/importance-of-dental-providers-role/pre-assessment.md [hunk 13]
```diff
@@ -4,11 +4,7 @@ date: 2019-05-13
...
-q1: ["SENTINAL_0", "SENTINAL_1", "SENTINAL_2", "SENTINAL_3", "SENTINAL_4"]
+q1: ["Strongly disagree", "Disagree", "Neither agree nor disagree", "Agree", "Strongly agree"]
 ---
-Foo
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
-
+How prepared do you feel as a dental clinician to address the overall health
```
Comment (sdreher) [inline content/domains/identify/importance-of-dental-providers-role/pre-assessment.md]: See above.

**Comment by nbuonin** [comment]
thanks @sdreher, I resolved your comments, and rebased

File: content/domains/identify/importance-of-dental-providers-role/post-assessment.md [hunk 14]
```diff
@@ -20,5 +20,5 @@ surgery made her feel much better. She asks you if by chance you could
 prescribe her hydrocodone because of your history with her and how comfortable
 she has become with you.
 
-How prepared do you feel as a dental clinician to address the overall health
-(including non-dental) of your patient population?
+I feel prepared as a dental clinician to address the overall health
+(including non-dental) of your patient population.
```
Comment (sdreher) [inline content/domains/identify/importance-of-dental-providers-role/post-assessment.md]: "my patient population"

Comment (nbuonin) [reply content/domains/identify/importance-of-dental-providers-role/post-assessment.md]: 🤦‍♂, got it.

**Comment by nbuonin** [comment]
@sdreher I think this is in better shape, but let's talk if you think it needs more.

**Pull request merged**